<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tcp</class>
 <widget class="QWidget" name="Tcp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="topMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QCheckBox" name="checkBox_validate_checksum">
     <property name="text">
      <string>Validate the TCP checksum</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_analyze_seq">
     <property name="text">
      <string>Analyze TCP sequence number</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_relative_seq">
     <property name="text">
      <string>Relative sequence number</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_ignore_TS_in_summery">
     <property name="text">
      <string>Ignore TCP timestamps in summery</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>152</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
